docs/changelog-fragments.d/.CHANGELOG-TEMPLATE.md.j2

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
This is the template Towncrier is set up to use. You can change virtually anything. Changing it won't affect the historic entries because Towncrier only adds new ones and doesn't touch anything that is already present in `CHANGELOG.md`.